mne/cov.py

```diff
@@ -1713,6 +1718,8 @@ def _estimate_rank_meeg_signals(data, info, scalings, tol=1e-4,
         thresholded to determine the rank are also returned.
     """
     picks_list = _picks_by_type(info)
+    if scalings is None:
+        scalings = DEFAULTS['scalings']
```

Comment:
make sure that if you call the function once with scalings not None and once with None you don't "remember" the first parameters passed. That would make behavior weird and un predictable.

I'll review when you confirm it works as expected.

thanks @adykstra